src/System.Private.CoreLib/src/System/GC.cs

```diff
@@ -50,15 +50,74 @@ public enum GCNotificationStatus
         NotApplicable = 4
     }
 
+    public struct GCMemoryInfo
```

Comment:
It looks like all the fields in this are readonly; can the struct be readonly, too?

Reply:
yes they are all readonly